h/test/renderers_test.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from pyramid.testing import DummyRequest
+
+from h import renderers
+
+def test_response_content_type():
+    renderer = renderers.CSV({})
+    req = DummyRequest()
+    renderer({}, { 'request': req })
```

Comment:
Only one *tiny* niggle here: good Python style doesn't have these extra spaces. You might want to install [`flake8`](https://pypi.python.org/pypi/flake8) to help catch things like this. If you run `flake8 h/test/renderers_test.py` that will tell you what needs fixing.